offchain/dispatcher/src/config.rs

```diff
@@ -76,6 +72,27 @@ pub struct DispatcherConfig {
     pub priority: Priority,
 }
 
+#[derive(Debug, Clone, Parser)]
+pub struct DispatcherHealthCheckConfig {
+    /// Enable health check
+    #[arg(long, env, default_value = "true")]
+    pub dispatcher_enable_healthcheck: String,
+
+    /// Host address of health check
+    #[arg(long, env, default_value = "0.0.0.0")]
+    pub dispatcher_healthcheck_address: String,
+
+    /// Port of health check
+    #[arg(long, env, default_value = "8082")]
+    pub dispatcher_healthcheck_port: u16,
+}
+
+#[derive(Debug)]
+pub struct Config {
```

Comment:
I think we can put the health check config inside DispatcherConfig without needing to create this Config struct.

Reply:
It was like that before, but in keeping with standardization, the indexer and advance runner had this layout. I'm just making them all look the same.